### PR TITLE
Windows rbe for Kokoro CI

### PR DESCRIPTION
**Pull request by billfeng327**
Script addition for Kokoro CI, passing build see https://source.cloud.google.com/results/invocations/2c1a3e3e-e69e-4368-a11b-e6caa8457330/targets

### CODE REVIEW

File: tools/internal_ci/windows/bazel_rbe.bat [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@rem Unless required by applicable law or agreed to in writing, software
+@rem distributed under the License is distributed on an "AS IS" BASIS,
+@rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@rem See the License for the specific language governing permissions and
+@rem limitations under the License.
+
+choco install bazel -y --version 0.23.2
+cd github/grpc
+set PATH=%PATH%;C:\python27\
+bazel --bazelrc=tools/remote_build/windows.bazelrc build :all --incompatible_disallow_filetype=false --google_credentials=%KOKORO_GFILE_DIR%/rbe-windows-credentials.json
```
Comment (jtattermusch) [inline tools/internal_ci/windows/bazel_rbe.bat]: future improvements: I think in order to get a "bazel" link for the build in Fusion UI, you'll need to do something like this: https://github.com/grpc/grpc/blob/d5769552c6db5dea7c6c7c4e3d8aa6a0ab5368e8/tools/internal_ci/linux/grpc_bazel_on_foundry_base.sh#L37

File: tools/internal_ci/windows/bazel_rbe.bat [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@rem
+@rem Unless required by applicable law or agreed to in writing, software
+@rem distributed under the License is distributed on an "AS IS" BASIS,
+@rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@rem See the License for the specific language governing permissions and
+@rem limitations under the License.
+
+choco install bazel -y --version 0.23.2
+cd github/grpc
+set PATH=%PATH%;C:\python27\
```
Comment (jtattermusch) [inline tools/internal_ci/windows/bazel_rbe.bat]: nit: to stay more consistent with  `prepare_build_widows.bat`, I'd do 
`set PATH=C:\Python27;%PATH%` instead. Fine to leave as is

https://github.com/grpc/grpc/blob/d5769552c6db5dea7c6c7c4e3d8aa6a0ab5368e8/tools/internal_ci/helper_scripts/prepare_build_windows.bat#L17

File: bazel/grpc_build_system.bzl [hunk 3]
```diff
@@ -112,10 +112,7 @@ def grpc_cc_library(
         visibility = visibility,
         testonly = testonly,
         linkopts = linkopts,
-        includes = [
-            "include",
-            "src/core/ext/upb-generated",
-        ],
+        includes = ["include"] + if_not_windows(["src/core/ext/upb-generated"]),
```
Comment (jtattermusch) [inline bazel/grpc_build_system.bzl]: nit: add a TODO?

**Comment by jtattermusch** [comment]
Known failures:
https://github.com/grpc/grpc/issues/18608
https://github.com/grpc/grpc/issues/15464

**Comment by jtattermusch** [comment]
I'm going to merge to get a passing windows RBE build on master.

File: bazel/grpc_build_system.bzl (hunk 3 above)
Comment (billfeng327) [reply bazel/grpc_build_system.bzl]: UPB is heavily reliant on POSIX and Linux headers, so it's not compatible with Windows in the near future.

File: tools/internal_ci/windows/bazel_rbe.bat (hunk 1 above)
Comment (billfeng327) [reply tools/internal_ci/windows/bazel_rbe.bat]: ack

File: tools/internal_ci/windows/bazel_rbe.bat (hunk 2 above)
Comment (billfeng327) [reply tools/internal_ci/windows/bazel_rbe.bat]: ack

**Pull request merged**